book/src/raw-block.md

```diff
@@ -6,7 +6,8 @@ Status | Min K8s Version | Max K8s Version | external-provisioner Version | exte
 --|--|--|--|--
 Alpha | 1.11 | 1.13 | 0.4 | 0.4
 Alpha | 1.13 | 1.13 | 1.0 | 1.0
-Beta | 1.14 | - | 1.1+ | 1.1+
+Beta | 1.14 | 1.18 | 1.1+ | 1.1+
+GA | 1.18 | - | - | -
```

Comment:
@jsafrane what version of external provisioner do you want to put here. 1.6? And attacher? Does it have any logic related to block volumes?

Reply:
* Attacher supports attaching block volumes since v1.1.0 and there was not any change in the code since then.
* The main provisioner code supports block since v1.1.0 and again, no changes/bugfixes there. Block support was gradually added to cloning and snapshots. I would say that v1.1+ is the most appropriate, as cloning and snapshots have their own feature.

Reply:
Thanks for bringing this up and thanks for the answer. Actually I was wondering which versions to pick, trying to check the changelog of the components but I couldn't find something related there. So now I updated the PR to `1.1+` also for the GA status..